### ISSUE TITLE
Add Year Income/Expenses

### ISSUE THREAD

**Issue by marsan27** [pmiddend/piggybudget#13]
Many people don't really care about how much they spent a day, but how much they spent during a whole month. Adding a whole year view divided into different months would be very useful

**Comment by pmiddend** [comment]
You're right, and I actually just implemented something like that. From version 2.4 on (should be built and released this week) you can see how much you spent each month (by category, and summed).
